src/app/account/settings/tabs/password-and-authentication-tab/components/admin-change-password/admin-change-password.component.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { ChangeDetectionStrategy, Component, inject, Input, OnInit } from "@angular/core";
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { ChangeAdminAccountPasswordFormType } from "../../password-and-authentication-tab.component.types";
+import { AccountService } from "src/app/account/account.service";
+import { matchFieldsValidator } from "src/app/common/helpers/data.helpers";
+import { ErrorSets } from "src/app/common/directives/form-validation-errors.types";
+
+@Component({
+    selector: "app-admin-change-password",
+    templateUrl: "./admin-change-password.component.html",
+    styleUrls: ["./admin-change-password.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AdminChangePasswordComponent implements OnInit {
+    @Input({ required: true }) public accountName: string;
+    public changeAdminAccountPasswordForm: FormGroup<ChangeAdminAccountPasswordFormType>;
+
+    private fb = inject(FormBuilder);
+    private accountService = inject(AccountService);
+    public readonly ErrorSets: typeof ErrorSets = ErrorSets;
+
+    public ngOnInit(): void {
+        this.changeAdminAccountPasswordForm = this.fb.nonNullable.group(
+            {
+                newPassword: ["", [Validators.required, Validators.minLength(8)]],
+                confirmPassword: ["", [Validators.required, Validators.minLength(8)]],
+            },
+            {
```

Comment:
Constant for length. Or even better: you could get it from the backend

Reply:
Extended UI configuration model